www/content/docs/environment-variables.mdx

```diff
@@ -86,3 +90,18 @@ const drupal = new DrupalClient(process.env.NEXT_PUBLIC_DRUPAL_BASE_URL, {
   },
 })
 ```
+
+## Optional
+
+These environment variables are **optional** and **not required as of `next-drupal 1.4.0`**.
+
+| Name                | Description                                                          |
+| ------------------- | :------------------------------------------------------------------- |
+| `DRUPAL_SITE_ID`    | The site id of the Next.js site on Drupal. Example: `marketing_site` |
```

Comment:
Would it be possible to also explain what this could be used for? The example for the `DRUPAL_FRONT_PAGE` env variable is very clear 👍

Reply:
Updated: https://next-drupal-git-feat-env-variables-chapter-three.vercel.app/docs/environment-variables#optional

I've also referenced the guide on filtering by site id.

Reply:
Perfect 👌 This makes it crystal clear what both of these env variables are used for. 💯